data/Record.js

```diff
@@ -43,20 +42,31 @@ export class Record {
      * requiring children to also be recreated.)
      *
      * @param {Object} c - Record configuration
+     * @param {Object} c.data - data for constructing the record.
      * @param {Object} c.raw - raw data for record.
```

Comment:
I know that apps aren't expected to construct records, but this is still a bit confusing here. How about:
```
c.data - data used to construct this record, pre-processed if applicable by store.processRawData().
c.raw - the same data, prior to any store pre-processing.
```
Would that be incrementally more clear?